flow/platforms/sky130hd/config.mk

```diff
@@ -150,3 +150,6 @@ export FILL_CONFIG = $(PLATFORM_DIR)/fill.json
 #
 # Template definition for power grid analysis
 export TEMPLATE_PGA_CFG ?= $(PLATFORM_DIR)/template_pga.cfg
+
+# OpenRCX extRules
+export RCX_RULES = ./$(PLATFORM_DIR)/rcx_patterns.rules
```

Comment:
Please remove leading `./` in case `$(PLATFORM_DIR)` is an absolute path.

Reply:
Fixed on 277610e